packages/core/lib/debug/printer.js

```diff
@@ -233,14 +233,14 @@ class DebugPrinter {
     this.config.logger.log(DebugUtils.formatCurrentInstruction(instruction));
 
     // printout 3 upcoming instructions
-    for (let i = Math.min(currentIndex + 1, instructions.length);
-      i <= currentIndex + upcomingInstructions;
+    for (let i = currentIndex + 1;
+      i < Math.min(currentIndex + upcomingInstructions + 1, instructions.length);
```

Comment:
I agree with @haltman-at's suggestion for clarity.

Reply:
agree, will make changes for clarity.  I will ping you tomorrow for the `prettier` comment.

Reply:
OK, the formatting thing -- basically, instead of 
```javascript
for (init;
  test;
  inc) {
}
```

it should be

```javascript
for (
  init;
  test;
  inc
) {
}
```

This also applies to `if`, etc -- if what goes inside the parentheses is long enough that it needs to be split into multiple lines, then the parentheses themselves don't go on the same lines as the content.

Again, sorry about this, `prettier` will enforce this in the future, etc.